package.json

```diff
@@ -23,7 +22,7 @@
     "format:js": "prettier --write '{packages,integration-tests,types}/**/*.{js,ts}'",
     "format:md": "prettier --parser markdown --write '**/*.md'",
     "test:coverage": "yarn build && npm test -- --no-watchman --coverage",
-    "test:ci": "yarn test -- --ci --no-watchman --maxWorkers=10",
+    "test:ci": "yarn test -u --ci --no-watchman --maxWorkers=10",
```

Comment:
Why the `-u` option? 🤔

Reply:
I initially used it to clear out stale snapshot tests, will remove it right away.